how-to/how-to-use-kata-containers-with-nemu.md

```diff
@@ -0,0 +1,59 @@
+# Kata Containers with NEMU
```

Comment:
Please could you add a table of contents to the start of the doc (ideally just after the title line for consistency with other docs).